engine/src/parameters.rs

```diff
@@ -493,6 +493,29 @@ impl TryFrom<JsonValue> for ResolveTransferCallArgs {
     }
 }
 
+pub struct FundEthAccountCallArgs {
+    pub address: Address,
+    pub amount: Wei,
+}
+
+impl TryFrom<JsonValue> for FundEthAccountCallArgs {
+    type Error = error::ParseTypeFromJsonError;
+
+    fn try_from(v: JsonValue) -> Result<Self, Self::Error> {
+        let string_address = &v.string("address")?;
+        if string_address.len() != 42 {
+            return Err(error::ParseTypeFromJsonError::Json(JsonError::InvalidString));
+        }
+        let address = Address::from_slice(&hex::decode(&string_address[2..string_address.len()]).unwrap());
```

Comment:
maybe instead of unwrapping return Error if it fails?

Reply:
Updated. Had to use the Errors without the additional details in order to simplify that code, since we need it just to test the integration with the bridge.